crates/tx5-core/src/file_check.rs

```diff
@@ -17,78 +18,85 @@ impl FileCheck {
     }
 }
 
-fn get_user_cache_dir() -> Option<std::path::PathBuf> {
-    match app_dirs2::app_root(
-        app_dirs2::AppDataType::UserCache,
-        &app_dirs2::AppInfo {
-            name: "host.holo.tx5",
-            author: "host.holo.tx5",
-        },
-    ) {
-        Ok(dir) => Some(dir),
-        _ => None,
+/// Get a path to a cache directory where the dependency Pion library can be written
+///
+/// Defaults to the UserCache directory returned by app_dirs2, specific to the target platform.
+/// Overridable via the env variable TX5_CACHE_DIRECTORY
+fn get_cache_dir() -> Result<std::path::PathBuf> {
```

Comment:
The env variable is only used for running tests on android, to override the directory to `/data/local/tmp`. 

Alternatively we could use `#[cfg(all(test, target_os="android"))]`, but for some reason it wasn't recognizing the target_os as android when running tests in CI. I couldn't figure it out.